strawberry/parent.py

```diff
@@ -8,31 +8,37 @@ class StrawberryParent: ...
 T = TypeVar("T")
 
 Parent = Annotated[T, StrawberryParent()]
-Parent.__doc__ = """Represents a parameter holding the parent resolver's value.
+"""Represents a parameter holding the parent resolver's value.
```

Comment:
Does this work as expected? I assume this is sufficient to get the docstring to show up in the documentation online, however, according to the following quick test code I ran, it looks like the `__doc__` property of `Parent` does not get set.

```python
from typing import Annotated


Parent = Annotated[int, dict(name="hi")]
"""My docstring

hi
"""

print(Parent.__doc__)
```

Reply:
it works with griffe 😊 https://strawberry-rocks-git-feature-api-docs-strawberry-graphql.vercel.app/api-reference/strawberry.Parent

Reply:
Oh, that's good to know. Are we okay with `help(Parent)` not showing the docstring in this and a few other cases?

Reply:
I'd say yes 😊